Frontend/src/components/AddAsset.jsx

```diff
@@ -77,15 +366,18 @@ function AddAsset({ onAdd }) {
 
       <div className="assets-list">
```

Comment:
This div is very fit to the whole page, it would be good if we have a margin around it.